src/lib/pages/account-details/index.tsx

```diff
@@ -120,8 +123,37 @@ const AccountDetailsBody = ({
     [accountAddress, hexAddress, navigate, tabParam]
   );
 
+  const { address } = useCurrentChain();
+  const { data, isLoading, isFetching } = useInitiaUsernameByAddress(
+    hexAddress,
+    move.enabled
+  );
```

Comment:
Can you add the specific name to the destructed response?

```ts
const { data: initiaUsernameData, isLoading: isInitiaUsernameDataLoading, isFetching: isInitiaUsernameDataFetching } = useInitiaUsernameByAddress(
    hexAddress,
    move.enabled
  );
```

Reply:
Also, can you add response interface type to `useInitiaUsernameByAddress`? Even though it just `string | null`, but we can use it across each components later.